src/rest/DependenciesCtrl.ts

```diff
@@ -60,6 +65,16 @@ export default class DependenciesCtrl {
         });
     }
 
+    private async getDependencyGraphData(req: restify.Request, res: restify.Response, next: restify.Next) {
+
+        let graphBuilder : FileDependencyGraphBuilder  = new FileDependencyGraphBuilder();
+        let urlBuilder : URLBuilder = new URLBuilder(AccessTokenManager.getGithubAccessToken());
+        let repoName = urlBuilder.getRepoName(req.query.url);
+        let data = await graphBuilder.getDependenciesFromProject("./data", repoName, req.query.start);
```

Comment:
This call can fail, do you want to wrap in a try-catch and log the error for debugging in case it does?

Reply:
updated!